src/Altinn.Platform/Altinn.Platform.PDF/src/main/resources/application.properties

```
@@ -1,8 +1,9 @@
 server.port=5070
-server.contextPath=/pdf
 spring.profiles.active=prod
-springfox.documentation.swagger.v2.path=/api/v1/swagger
-logging.level.root=WARN
+springdoc.api-docs.path=/api/v1/swagger
+springdoc.packagesToScan=altinn.platform.pdf
+springdoc.pathsToMatch=/api/**
+logging.level.root=DEBUG
```

Change root log level to WARN again
```suggestion
logging.level.root=WARN
```